docs/src/content/en/guides/build-your-ui/ai-sdk-ui.mdx

```diff
@@ -47,9 +47,54 @@ The recommended way of using Mastra and AI SDK together is by installing the `@m
   </TabItem>
 </Tabs>
 
-### `chatRoute()`
+The `@mastra/ai-sdk` route handlers integrate directly with AI SDK UI's hooks. AI SDK UI provides three main hooks: [`useChat()`](https://ai-sdk.dev/docs/ai-sdk-ui/chatbot), [`useCompletion()`](https://ai-sdk.dev/docs/ai-sdk-ui/completion), and [`useObject()`](https://ai-sdk.dev/docs/ai-sdk-ui/object-generation).
 
-When setting up a [custom API route](/docs/v1/server-db/custom-api-routes), use the `chatRoute()` utility to create a route handler that automatically formats the agent stream into an AI SDK-compatible format.
+If you haven't already, install the required AI SDK packages:
```

Comment:
Maybe move the installation of ⁠ai and ⁠@ai-sdk/react before the ⁠@mastra/ai-sdk step, since they function as dependencies/requirements for ⁠@mastra/ai-sdk. 

You can consider combining everything into a single install command so users can copy–paste one snippet instead of multiple steps:

```
pnpm i ai @ai-sdk/react @mastra/ai-sdk
```

My understanding is that, if the user already has any of these packages installed, rerunning the command is safe and will not change anything.